test/REXUniswapV3Market.test.ts

```diff
@@ -1515,5 +1515,24 @@ describe('REXUniswapV3Market', () => {
         })
         .exec(bobSigner)
     })
+
+    it('#4.2 Should return the correct next distribution time', async () => {
+      const gasPrice = 32000 // 32000 GWEI
+      const gasLimit = 12000000
+      const tokenToMaticRate = 10 ** 9 // 1 matic = 1 usd
+      const lastDistributedAt = await market.lastDistributedAt()
+
+      const netFlowRate = await sf.cfaV1.getNetFlow({
+        superToken: ricochetMATICx.address,
+        account: market.address,
+        providerOrSigner: adminSigner,
+      })
+
+      console.log(netFlowRate.toString())
+
+      const actualDistributionTime = await market.getNextDistributionTime(gasPrice, gasLimit, tokenToMaticRate)
+
+      expect(actualDistributionTime).to.greaterThan(lastDistributedAt)
```

Comment:
can this be more precise like:
```
      expect(actualDistributionTime).to.equal(lastDistributedAt + gasPrice * gasLimit * tokenToMaticRate / inflowRate)
```

Reply:
Done👍, Added `calculatedDistributionTime` instead of suing `greaterThan`.